CHANGELOG.md

```diff
@@ -13,7 +13,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- `tcell` dependency was upgraded 1.4.0.
+- `tcell` dependency was upgraded v2.0.0.
```

Comment:
(nit) I made a typo here, missing "to" in ... to v2.0.0 in case you don't mind fixing it.